src/main/java/expiryeliminator/data/RecipeList.java

```diff
@@ -53,4 +53,28 @@ public boolean contains(Recipe recipe) {
     public int size() {
         return recipes.size();
     }
+
+    /**
+     * Returns a string representation of the whole recipe list.
+     *
+     * @return returns a string representing all the recipes.
+     */
+    public String printWholeRecipeList() {
```

Comment:
I suggest renaming this to `getWholeRecipeList()` or something else, because it doesn't actually print out the recipe here, it only returns the string of the recipe list.